language/de/common.php

```diff
@@ -822,7 +822,7 @@
 	'USERS'					=> 'Benutzer',
 	'USE_PERMISSIONS'		=> 'Berechtigungen des Benutzers testen',
 
-	'USER_NEW_PERMISSION_DISALLOWED'	=> 'Leider bist du nicht berechtigt, diese Funktion zu nutzen. Du hast dich vermutlich erst vor kurzem registriert und musst dich noch mehr im Board beteiligen, damit du diese Funktion nutzen kannst.',
+	'USER_NEW_PERMISSION_DISALLOWED'	=> 'Leider bist du nicht berechtigt, diese Funktion zu nutzen. Du hast dich vermutlich erst vor Kurzem registriert und musst dich noch mehr an Diskussionen im Board beteiligen, damit du diese Funktion nutzen kannst.',
```

Comment:
Ich hatte es für einen Tippfehler gehalten und geändert, weil ich eh in der Zeile war.